index.js

```diff
@@ -26,10 +26,16 @@ module.exports = (function () {
    * @return {Promise} returns String passage
    */
   bible.get = function (psg, ver) {
-    if (typeof psg !== 'string') { throw new TypeError('Expected a string'); }
+    if (typeof psg !== 'string') {
+      throw new TypeError('Passage should be a string');
+    }
+    if (typeof ver !== 'undefined' && typeof ver !== 'string') {
+      throw new TypeError('Version should be a string');
+    }
 
     var self = this;
     var v = ver || 'asv';
```

Comment:
You can also call `toLowerCase` on `ver` to avoid re-assigning `v`. I'd feel more comfortable if you did that.